pubspec.yaml

```diff
@@ -12,5 +12,5 @@ dependencies:
   path: '>=0.9.0 <2.0.0'
 dev_dependencies:
   benchmark_harness: '^1.0.4'
-  scheduled_test: '^0.12.0'
-  test: '^0.12.18+1'
+  test: '^0.12.29+1'
```

Comment:
Why `+1`? Lower bounds should generally be pinned to versions that introduce new features.

Reply:
Pinning to the latest version is always *safe*, but it's not usually the best thing to do as a responsible member of the package ecosystem. You want to make the version range as wide as possible while guaranteeing that the features your package uses are available to minimize version churn and the possibility of version lock.